plugin/plugin_manager.go

```diff
@@ -29,12 +29,10 @@ type IPluginManager interface {
 }
 
 func NewPluginManager() *PluginManager {
-	pluginLoggerOpt := &hclog.LoggerOptions{
-		Name:   "plugin-manager",
-		Output: os.Stdout,
-		Level:  hclog.Info,
-	}
-	logger := hclog.New(pluginLoggerOpt)
+	logger := log.NewLogrus(
```

Comment:
Why are we constructing a new logger and not injecting the one available in application ?

Reply:
Ack, missed this. This will be used on client level, so that it should be use the log level that comes from client config